x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps/handlers.ts

```diff
@@ -5,24 +5,44 @@
  * 2.0.
  */
 
-import type { RequestHandler } from 'kibana/server';
+import type { KibanaResponseFactory, RequestHandler, IKibanaResponse, Logger } from 'kibana/server';
 import type { SecuritySolutionRequestHandlerContext } from '../../../types';
 
 import { ExceptionListClient } from '../../../../../lists/server';
 
 import {
   DeleteTrustedAppsRequestParams,
+  GetOneTrustedAppRequestParams,
   GetTrustedAppsListRequest,
   PostTrustedAppCreateRequest,
+  PutTrustedAppsRequestParams,
+  PutTrustedAppUpdateRequest,
 } from '../../../../common/endpoint/types';
+import { parseExperimentalConfigValue } from '../../../../common/experimental_features';
+import { EndpointAppContext } from '../../types';
 
 import {
   createTrustedApp,
   deleteTrustedApp,
+  getTrustedApp,
   getTrustedAppsList,
   getTrustedAppsSummary,
-  MissingTrustedAppException,
+  updateTrustedApp,
 } from './service';
+import { TrustedAppNotFoundError, TrustedAppVersionConflictError } from './errors';
+
+const getBodyAfterFeatureFlagCheck = async (
+  body: PutTrustedAppUpdateRequest | PostTrustedAppCreateRequest,
+  endpointAppContext: EndpointAppContext
+): Promise<PutTrustedAppUpdateRequest | PostTrustedAppCreateRequest> => {
+  const config = await endpointAppContext.config();
+  const isTrustedAppsByPolicyEnabled = parseExperimentalConfigValue(config.enableExperimental)
```

Comment:
Wondering if we should avoid the constant parsing and instead do it once in the `EndpointAppContext` and expose it there. Doing the parsing there would hepefully avoid having this function here be `async` as well, since we would not have to retrieve the `.config()` again. 

Thoughts?

Reply:
You mean move this into x-pack/plugins/security_solution/server/plugin.ts and add the parsed enableExperimental on the EndpointAppContext type? 
Make sense for me since it will be available everywhere with the context,

Reply:
yes, thats what I mean. I think that might be ok because we're not using the Observable approach to sync up with the config.